seqr/models.py

```diff
@@ -1011,19 +1011,36 @@ def __unicode__(self):
     def _compute_guid(self):
         return 'VSR%07d_%s' % (self.id, _slugify(str(self)))
 
-class DeletableSampleMetadataModel(models.Model):
 
-    sample = models.ForeignKey('Sample', on_delete=models.CASCADE, db_index=True)
-    gene_id = models.CharField(max_length=20)  # ensembl ID
+class BulkOperationBase:
+    @classmethod
+    def bulk_create(cls, user, new_models, parent=None):
+        """Helper bulk create method that logs the creation"""
+        for model in new_models:
+            model.created_by = user
+        models = cls.objects.bulk_create(new_models)
+        log_model_bulk_update(logger, models, user, 'create', parent=parent)
+        return models
 
     @classmethod
-    def bulk_delete(cls, user, queryset=None, **filter_kwargs):
+    def bulk_delete(cls, user, queryset=None, parent=None, **filter_kwargs):
```

Comment:
per my previous review, I do not think we should support "parent" here. That should be a different helper function if need be